pages/linux/nmcli-connection.md

```diff
@@ -0,0 +1,27 @@
+# nmcli connection
+
+> Connection management with NetworkManager.
+
+- List all NetworkManager connections (shows name, uuid, type and device):
+
+`nmcli connection`
+
+- Activate a connection by specifying an uuid:
+
+`nmcli connection up uuid {{uuid}}`
+
+- Deactivate a connection:
+
+`nmcli connection down uuid {{uuid}}`
+
+- Create a static IPv4-only connection:
```

Comment:
Perhaps we could alter the order of these 3 examples to be more similar to the order people might use them in. What about reversing it, like this?

 - Create an auto-configured.....
 - Static IPv6
 - Static IPv4

Reply:
I am coming from enterprise datacenters world, I can tell you that IPv6 is not a thing there yet. And at homes, that's disaster. But sure, won't hurt, shaked it a bit.

Reply:
I see. With IPv6 being the future, it seems like a sensible idea to emphasise that in tldr-pages.